src/presentation/components/atoms/loader/index.tsx

```
@@ -0,0 +1,7 @@
+import { Content } from "./styles";
+
+function Loader({}) {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)